### PR TITLE
Explained login to docker container

### LINKED ISSUE #125: Explain login to docker container
Ticket  #78 requested to describe persistent storage of notebook files and credential store.

The current ticket requests to enhance the user guide for AI-Lab Docker edition `doc/user_guide/docker/docker_usage.md` to describe how to login to a docker container. 

This is especially important for the instructions to change the initial password of the Jupyter server.
Changing the initial password affects the following files 
* [doc/user_guide/docker/docker_usage.md](https://github.com/doc/user_guide/docker/docker_usage.md)
* [exasol/ds/sandbox/runtime/ansible/roles/jupyter/templates/etc/update-motd.d/999-jupyter](https://github.com/exasol/ai-lab/tree/main/exasol/ds/sandbox/runtime/ansible/roles/jupyter/templates/etc/update-motd.d/999-jupyter)

### PR DESCRIPTION
**Pull request by ckunki**
closes #125

### CODE REVIEW

File: doc/user_guide/docker/docker_usage.md [hunk 1]
```diff
@@ -142,10 +142,35 @@ port to the same port then you can connect with http://localhost:49494.
 └─┘┴  ─┴┘┴ ┴ ┴ └─┘   ┴ └─┘└─┘┴└─  └┘└─┘┴   ┴  ┴ └─┘┴└─  ┴  ┴ ┴└─┘└─┘└┴┘└─┘┴└──┴┘ o
 
 The default password is "ai-lab".
-To update the password as user root run
+To update the password login to the Docker container as user root and run
```
Comment (ahsimb) [inline doc/user_guide/docker/docker_usage.md:145]: "as a root user" or "the user root"

File: doc/user_guide/docker/docker_usage.md [hunk 2]
```diff
@@ -142,10 +142,35 @@ port to the same port then you can connect with http://localhost:49494.
...
     /root/jupyterenv/bin/jupyter-lab server password
 ```
 
 Using an internet browser you then can connect to the Jupyter server running in the Docker container in order to follow the tutorials presented by a set of Jupyter notebooks, see [Connecting to Jupyter Service](../user_guide.md#connecting-to-jupyter-service).
 
 For parameter `<host>`: If your daemon machine is identical to the machine your browser is running on then you can replace `<host>` by `localhost` otherwise please use the IP address of the daemon machine.
+
+The following section explains how to login to the Docker container in order to change the default password for example.
```
Comment (ahsimb) [inline doc/user_guide/docker/docker_usage.md:153]: Maybe "The following section explains how to login to the Docker container to change settings, such as the default password".

File: doc/user_guide/docker/docker_usage.md [hunk 3]
```diff
@@ -142,10 +142,35 @@ port to the same port then you can connect with http://localhost:49494.
...
+
+## Login to the Docker container
+
+To update the password you first need to login to the Docker container.
```
Comment (ahsimb) [inline doc/user_guide/docker/docker_usage.md:157]: I would rephrase it a bit. The way it's written it may seem like you should first login to the Docker container and then run the below command.

To update the password you should log in to the Docker container.

First, you need to find out the container's ID.

The following command shows ...

File: doc/user_guide/docker/docker_usage.md [hunk 4]
```diff
@@ -142,10 +142,35 @@ port to the same port then you can connect with http://localhost:49494.
...
+```
+
+Here is a sample output
+
+```
+CONTAINER ID   IMAGE     COMMAND        NAMES
+1199447716d4   image:2   "entrypoint"   funny_rabbit
+```
+
+The following command enables you to login as user `root` to the specified container:
```
Comment (ahsimb) [inline doc/user_guide/docker/docker_usage.md:172]: the user

File: exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py [hunk 5]
```diff
@@ -100,7 +100,7 @@ def exit_on_error(rc):
         └─┘┴  ─┴┘┴ ┴ ┴ └─┘   ┴ └─┘└─┘┴└─  └┘└─┘┴   ┴  ┴ └─┘┴└─  ┴  ┴ ┴└─┘└─┘└┴┘└─┘┴└──┴┘ o
 
         The default password is "{password}".
-        To update the password as user {user} run
+        To update the password login to the Docker container as user {user} and run
```
Comment (ahsimb) [inline exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py:103]: "log in", "login" is a noun.
the user.

**Review by ahsimb — CHANGES_REQUESTED** [review]

File: doc/user_guide/docker/docker_usage.md (hunk 1 above)
Comment (ckunki) [reply doc/user_guide/docker/docker_usage.md:145]: I will pick the latter, as the word "root" is calculated dynamically and ticket #66 requests to change it in the future.

**Review by ahsimb — APPROVED** [review]
